pkg/runtime/runtime.go

```diff
@@ -682,7 +682,11 @@ func (a *DaprRuntime) sendBindingEventToApp(bindingName string, data []byte, met
 		}
 
 		if err != nil {
-			return nil, errors.Wrap(err, "error invoking app")
+			body := resp.Data
+			if len(body) > 75 {
```

Comment:
Why 75? This is called "magic number". There should be a rationale behind it and be moved to a constant if this really needs to be 75.

Reply:
My reasoning for that was to keep the error message to 100 characters max since the existing error already had 25 characters. If that 100 char limit should be removed or increased, let me know.